### PR TITLE
(PC-8860) Change wording + redirection to website

### PR DESCRIPTION
**Pull request by emmajmn**
Link to JIRA ticket: https://passculture.atlassian.net/browse/PC-8860

## Checklist

I have:

- [x] Made sure the title of my PR follows the convention `[PC-XXX] <summary>`.
- [x] Made sure my feature is working on the relevant real / virtual devices.
- [x] Written **unit tests** for my feature.
- [ ] Written **documentation on Notion** for my feature.
- [ ] Added new reusable components to the AppComponents page and communicate to the team on slack
- [x] Added a **screenshot** for UI tickets.
- [ ] Attached a **ticket number** for any added TODO/FIXME \
       (for tech tasks, give the best context about the TODO resolution: what/who/when).

## Screenshots

| \*iOS - [Phone name] |
| -------------------- |
| image | 

## Deploy hard

If native code (ios/android) was modified, **after** the PR is merged, on the master branch, upgrade the **app version** (+1 patch):

- if you want an hard deployment of the testing environment, use `yarn trigger:testing:deploy`

### CODE REVIEW

File: src/features/auth/support.services.ts [hunk 1]
```diff
@@ -8,9 +8,9 @@ export const contactSupport = {
       analytics.logMailTo('forGenericQuestion')
     )
   },
-  forSignupConfirmationEmailNotReceived(email: string) {
+  forSignupConfirmationEmailNotReceived() {
     openExternalUrl(
-      `mailto:${env.SUPPORT_EMAIL_ADDRESS}?subject=Non-r%C3%A9ception%20d'email%20de%20confirmation%20de%20compte&body=Bonjour%2C%0D%0A%0D%0AJe%20vous%20%C3%A9cris%20afin%20de%20vous%20informer%20de%20la%20non-r%C3%A9ception%20de%20l'email%20de%20confirmation%20de%20compte%20associ%C3%A9%20%C3%A0%20l%E2%80%99adresse%20e-mail%20${email}%2C%20y%20compris%20dans%20mes%20spams.%0D%0A%0D%0ABien%20cordialement%2C`,
+      'https://aide.passculture.app/fr/articles/5257121-je-n-ai-pas-recu-le-mail-de-confirmation-de-creation-de-compte',
```
Comment (blixit) [inline src/features/auth/support.services.ts:13]: le lien en dur dans le code ?

File: src/features/auth/signup/SignupConfirmationEmailSent.tsx [hunk 2]
```diff
@@ -54,12 +54,18 @@ export const SignupConfirmationEmailSent: FunctionComponent<Props> = ({ route })
...
             <Typo.Body>
-              {t`L'e-mail peut prendre quelques minutes à arriver. Pense à vérifier tes spams !`}
+              {t`Nous devons vérifier tes informations : l'email peut prendre quelques minutes pour arriver.`}
             </Typo.Body>
           </CenteredText>
           <Spacer.Column numberOfSpaces={5} />
-          <Typo.Body>{t`Si l'e-mail n'arrive pas, tu peux : `}</Typo.Body>
-          <ButtonTertiary title={t`Contacter le support`} onPress={onContactSupport} icon={Email} />
+          <CenteredText>
+            <Typo.Body>{t`Si tu rencontres des difficultés, tu peux consulter notre centre d'aide.`}</Typo.Body>
```
Comment (blixit) [inline src/features/auth/signup/SignupConfirmationEmailSent.tsx:62]: t'as pensé à régénérer les traductions ?

Comment (emmajmn) [reply src/features/auth/signup/SignupConfirmationEmailSent.tsx:62]: nope j'ajoute ça !

File: src/features/auth/support.services.ts (hunk 1 above)
Comment (emmajmn) [reply src/features/auth/support.services.ts:13]: tu as une meilleure reco ? :)

**Review by blixit — APPROVED** [review]

**Pull request merged**